plugins/GSdx/GSState.cpp

```diff
@@ -3413,7 +3413,7 @@ bool GSC_Spartan(const GSFrameInfo& fi, int& skip)
 {
 	if(skip == 0)
 	{
-		if(g_crc_region == CRC::NoRegion &&fi.TME && fi.FBP == 0x02000 && fi.FPSM == PSM_PSMCT32 && fi.TBP0 == 0x00000 && fi.TPSM == PSM_PSMCT32)
+		if(g_crc_region == CRC::EU &&fi.TME && fi.FBP == 0x02000 && fi.FPSM == PSM_PSMCT32 && fi.TBP0 == 0x00000 && fi.TPSM == PSM_PSMCT32)
```

Comment:
the former region value of that game at GSCrc.cpp was NoRegion after updating the region to EU. he also had to change this to EU to make the CRC hack accessible for the PAL version. if this isn't changed to EU then, the CRC hack will not be accessible for the PAL version.